apps/fabric-website/src/SiteDefinition/SiteDefinition.pages/Styles/web.tsx

```diff
@@ -174,9 +174,9 @@ export const stylesPagesWeb: INavPage[] = [
       ),
   },
   {
-    title: 'Fabric 7 Updates',
+    title: 'Fluent UI 7 Updates',
```

Comment:
This should probably remain 'Fabric 7 Updates' since the content of the page is about Fabric 7. 
We could also just remove the page since "what's new in fabric 7" isn't super relevant.

Reply:
btw, here's a link to the page (I think): https://developer.microsoft.com/en-us/fabric#/styles/web/fabric-7